src/assets/stylesheets/index.scss

```diff
@@ -109,6 +109,14 @@ a:focus-visible,
   transition: all ease-in-out 0.3s;
 }
 
+[data-scrivito-editing] {
+  a:focus-visible,
+  .btn:focus-visible,
+  :focus-visible {
+    outline: none !important;
```

Comment:
* [ ] This disables the focus style completely, which defeats the color picker input styles. If this is intended, we should delete these styles.